{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/templates/frontend/wysiwyg.html

```diff
@@ -0,0 +1,82 @@
+{% extends 'frontend/_base.html' %}
+
+{% block main %}
+  <div class="wys-WYSIWYG" style="max-width: 800px">
+    <p class="wys-Standfirst">Standfirst lorem ipsum dolor sit amet, consectetur adipiscing elit.</p>
```

Comment:
Can we include the class `wys-Standfirst` in `wysiwyg.css` even if it just has `color: var(--Colour_Something)`